charts/simple-app/values.yaml

```diff
@@ -487,6 +487,11 @@ datadog:
   metricsToScrape:
     - '"*"'
 
+  # -- (`bool`) If true, then it will enable application logging to datadog.
+  # To enable datadog logging, datadog.service value must be set as it's
+  # being used as source and service name in datadog logging configuration.
```

Comment:
Since we use “scrapeMetrics”, how about “scrapeLogs” or “collectLogs”?

Reply:
Changed to scrapeLogs